test/deploy/kubernetes/3rdparty-services/component-packages/symas_openldap.yaml

```diff
@@ -16,7 +16,7 @@ spec:
         - "/opt/entrypoint.sh"
       args:
         - "--rootdnpassword"
-        - "seagate2"
+        - "seagate1"
```

Comment:
Why seagate1 ? Can we say root_dn_password ?

Reply:
Yes we can change it.